config/config.default.php

```diff
@@ -276,6 +276,9 @@
     // Enables the T-Shirt configuration on signup and profile
     'enable_tshirt_size'      => (bool) env('ENABLE_TSHIRT_SIZE', true),
 
+    // When true changes everything from shirts to goodies and disables shirt size
+    'other_goodie'            => (bool) env('OTHER_GOODIE', true),
```

Comment:
Having T-Shirts enabled by default would make more sense?
```suggestion
    'other_goodie'            => (bool) env('OTHER_GOODIE', false),
```